style-guide.md

```diff
@@ -0,0 +1,111 @@
+* Assume Linux users are using Ubuntu or know how to translate into their own distro
+	* Exception: If directing very new users to install software via apt-get, consider mentioning that the instructions are specific to Ubuntu -- UCSC's linux lab for students runs on CentOS (which uses yum instead) so this is not a rare occurrence
+* Keep in mind that some of our users are using Macs, so make a note about these situations in the text if they occur:
+	* If there is an important functional difference between the Linux and the Mac version, such as xargs
+	* If there is an important installation difference between the Linux and the Mac version, such as the installation of the Dockstore CLI
+	* If something does not work/works differently on Apple Silicon Macs
+	* If there is an important difference between the bash and zsh version of something
+		* Do not assume that a Mac user is aware of whether they are using bash or zsh (the default changed in Catalina) or knows how to change between the two
+		* Do not worry about other shells
+
+## Coding in RST
+* Write in RST, not markdown
+* Use `.. comment` to write comments
+* Use consistent indentation spacing within a single RST file
+* Do not break up body text with a newline every x characters
+* Use \`this style \<https://example.com\>_` of formatting external hyperlinks, as it is less error prone than the method involving setting the target in another paragraph
+* Use headings in this way throughout the entire repository to avoid issues with embedding RST files into other RST files:
+
+> `page title`  
+> `==========`
+> 
+> `heading level 1`  
+> `---------------`
+>
+> `heading level 2`  
+> `~~~~~~~~~~~~~~~`
+>
+> `heading level 3`  
+> `***************`
+>
+> `heading level 4`  
+> `+++++++++++++++`
+>
+> `heading level 5`  
+> `\``````````````
+>
+ 
+## International
+See also: https://developers.google.com/style/translation 
+
+* Avoid using idioms ("it's raining cats and dogs in Santa Cruz")
+* Avoid using culturally-specific comparisons ("the button is green like a dollar bill")
+* Try to use relatively simple English whenever possible
+* Make sure programs/templates you are providing to the user have a way to handle non-ASCII names and names with apostrophes and hyphens
+
+## Inclusion
+See also: https://developers.google.com/style/inclusive-documentation
+
+* Use gender-neutral language wherever possible (firefighter vs fireman)
+* Use gender-neutral pronouns when referring to a hypothetical person ("they" instead of "he or she"), even if that requires changing a verb conjugation ("they **adjust** their settings" vs "he or she **adjusts** his or her settings")
+* Avoid terms like master/slave or blacklist/whitelist - use parent/child or denylist/allowlist
```

Comment:
Avoid using socially-charged terms for technical concepts. For example, avoid terms such as blacklist, native feature, and first-class citizen. Instead, consider allow/denylist, built-in feature, core feature.

Reply:
Hmmm, has "native" been cancelled? 
This is pretty common for AWS, etc. 
https://aws.amazon.com/blogs/opensource/cloud-native-computing/
https://docs.microsoft.com/en-us/dotnet/architecture/cloud-native/definition

Reply:
there's a bit of discussion on it here, generally, just try to avoid when you can: https://developers.google.com/style/inclusive-documentation

Reply:
How do you feel about https://github.com/dockstore/dockstore-documentation/pull/197/commits/0b734fd5b59310acc5ab97c08c71274ee74dbddd?

Reply:
Sure